### PR TITLE
[s] Gives hydroponics a toy to play with. Wear safety glasses while using.

### PR DESCRIPTION
**Pull request by Supermichael777**
:cl: 
rscadd: Hydroponics has gotten a new toy so they can have fun while the plants grow.
rscadd: Please ensure you feed the ammunition before storage.
/:cl:

### CODE REVIEW

**Comment by Zilenan91** [comment]
This is something that could probably go in the actual game

File: code/modules/projectiles/guns/projectile/shotgun.dm [hunk 1]
```diff
@@ -156,6 +156,28 @@
...
+	icon_state = "762-casing"
+	caliber = "Angrybee"
+	projectile_type = /obj/item/projectile/bullet/beebee
+	
+/obj/item/projectile/bullet/beebee
+	damage = 5
+
+/obj/item/projectile/bullet/beebee/on_hit(atom/target, blocked = 0)
+	..()
+	new /mob/living/simple_animal/hostile/poison/bees(target.loc)
```
Comment (KazeEspada) [inline code/modules/projectiles/guns/projectile/shotgun.dm]: use get_turf(target) also, set the new bees target to the target if its living.

Comment (Supermichael777) [inline code/modules/projectiles/guns/projectile/shotgun.dm]: Holy into mobcode and i cannot find where get_turf is documented

Comment (KazeEspada) [inline code/modules/projectiles/guns/projectile/shotgun.dm]: Okay, get_turf() recursively gets the loc until it gets a turf.  Also, setting target is as easy as first istyping() the target(to make sure its a mob), then typecast a var to bees, bees.target = target.

**Comment by Iamgoofball** [comment]
:+1: :x: :100:

**Comment by RemieRichards** [comment]
Yeah no this can actually go in the game, Bee weaponry is planned for after the main pr is merged (I was thinking of Bee grenades, but a shotgun works too)

:+1:

**Comment by Bawhoppen** [comment]
remie I got just the sprite for bee grenades for you

**Comment by RemieRichards** [comment]
niiice.

**Comment by AlexCrimson** [comment]
Is it similar to the Beenade sprite/whatever in Terraria? It better... bee.

**Comment by PKPenguin321** [comment]
>Is it similar to the Beenade sprite/whatever in Terraria? It better... bee.


bruh

**Comment by Supermichael777** [comment]
I don't think its currently compatible with reagent bees, best way to do that would be to use a pneumatic cannon sub-type because bullets and casings are hard-coded only to work with /projectile/.  I tried actually firing the bees and had no luck.

Eggs would practical have to be designed around being shot from the cannon too if you want reagent bees to retain the reagent.

This was a bad pun gone wrong plus a dr. mcninja reference.

**Comment by Iamgoofball** [comment]
no, this looks compatible

**Comment by Chocobro** [comment]
giving permission to add this to the main game

**Pull request merged**